client/src/store/slices/users.slice.js

```diff
@@ -0,0 +1,34 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const usersSlice = createSlice({
+  name: 'users',
+  initialState: {
+    usersList: [],
+    user: null,
+  },
+  reducers: {
+    initUsersList(state, action) {
+      state.usersList = action.payload.usersList;
+    },
+    initUser(state, action) {
+      state.user = action.payload.user;
+    },
+    changeUser(state, action) {
```

Comment:
why and for what this action ? 
probably you could get the user from server request , and not from the userList
also, probably you will fetch userList by pagination or specific users, and not all users on you db 
the userList is for display user list mode, not to describe you all users in your db 
so, the action to select user by user id is not realistic

Reply:
so, remove this action please

Reply:
ok removed